hal/common/retarget.cpp

```diff
@@ -713,6 +713,39 @@ extern "C" void __env_unlock( struct _reent *_r )
 {
     __rtos_env_unlock(_r);
 }
+
+extern "C" int __cxa_guard_acquire (uint64_t *guard_object_p)
```

Comment:
@c1728p9 Where did you take these prototypes from ? ``(uint64_t *guard_object_p)`` ?